text2text/identifier.py

```diff
@@ -0,0 +1,50 @@
+import os
+import requests
+import pickle
+import text2text as t2t
+
+
+class Identifier():
+
+    def _get_model(self):
+        """Download and loads model from google drive"""
+
+        file_id = "10-YNx9yDVJTUijVz_2aopcIF8rdXR78R"
+        URL = "https://docs.google.com/uc?export=download"
+        session = requests.Session()
+        response = session.get(URL, params = { 'id' : file_id }, stream = True)
+        for key, value in response.cookies.items():
+            if key.startswith('download_warning'):
+                params = { 'id' : file_id, 'confirm' : value }
+                response = session.get(URL, params = params, stream = True)
+
+        CHUNK_SIZE = 32768
+
+        model_file = "identifier.pkl"
+        if not os.path.exists(model_file):
```

Comment:
Can we check if the file exist in the beginning of the method?

Reply:
Yes, that should have been the right thing to do, my bad :)